### PR TITLE
testing/openblas: upgrade to 0.2.18 and improve abuild

### PR DESCRIPTION
**Pull request by jirutka**

### CODE REVIEW

File: testing/openblas/APKBUILD [hunk 1]
```diff
@@ -10,53 +11,52 @@ license="BSD"
...
 # NO_AFFINITY: don't expect to work on musl, also breaks R
 # NUM_THREADS=32: match CONFIG_NR_CPUS
-FLAGS="USE_OPENMP=0 NO_LAPACK=0 NUM_THREADS=32 MAJOR_VERSION=3 \
-	NO_AFFINITY=1 DYNAMIC_ARCH=$DYNAMIC_ARCH"
+_flags="USE_OPENMP=0
+	NO_LAPACK=0
+	NUM_THREADS=32
+	MAJOR_VERSION=3
+	NO_AFFINITY=1
+	DYNAMIC_ARCH=0
```
Comment (tkelman) [inline testing/openblas/APKBUILD]: dynamic arch will give much better performance on recent processors

Comment (jirutka) [inline testing/openblas/APKBUILD]: Oh, thanks a lot for noticing it! This change isn’t intentional, I’ve accidentally commited wrong changes.

Comment (tkelman) [inline testing/openblas/APKBUILD]: looked like you might have wanted to use your new `_dynamic_arch` var

**Comment by jirutka** [comment]
Merged in 19836dc2130d94520c60c31a59a0379da8453e28.

**Pull request closed without merging**